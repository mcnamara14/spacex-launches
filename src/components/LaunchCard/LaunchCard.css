.LaunchCard {
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 33px;
  padding: 15px 20px;
  background:rgba(255, 255, 255, 0.7);
  border-radius: 5px;
  border: 1px solid #fff;
}

.LaunchCard div {
  margin-right: 30px;
  padding: 0 5px;
  text-align: left;
  align-items: center;
  color: #444444;
  font-size: 14px;
  display: flex;
}

.LaunchCard .badge, .LaunchCard .id, .LaunchCard .article {  
  justify-content: center;
}