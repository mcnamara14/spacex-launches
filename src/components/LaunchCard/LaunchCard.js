import React, { Component } from 'react';
import './LaunchCard.css';

class LaunchCard extends Component {
  render() {
    return (
      <section className="LaunchCard" >
        
      </section>
    );
  }
}

export default LaunchCard;
