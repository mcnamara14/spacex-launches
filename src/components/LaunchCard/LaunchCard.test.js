import React from 'react';
import LaunchCard from './LaunchCard';
import { shallow } from 'enzyme';

describe('LaunchCard', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<LaunchCard />);

    expect(wrapper).toMatchSnapshot();
  });
});