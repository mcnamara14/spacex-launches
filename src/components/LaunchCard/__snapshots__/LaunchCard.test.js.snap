// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LaunchCard should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LaunchCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="badge"
>
          <img
                    alt="flight badge"
                    src={undefined}
          />
</div>,
        <div
          className="name"
/>,
        <div
          className="type"
/>,
        <div
          className="date"
/>,
        <div
          className="details"
/>,
        <div
          className="id"
/>,
        <div
          className="article"
>
          <a
                    href={undefined}
                    target="_blank"
          >
                    <img
                              alt="article link"
                              src="link.svg"
                    />
          </a>
</div>,
      ],
      "className": "LaunchCard",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="flight badge"
            src={undefined}
/>,
          "className": "badge",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "flight badge",
            "src": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "name",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "type",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "date",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "details",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "id",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href={undefined}
            target="_blank"
>
            <img
                        alt="article link"
                        src="link.svg"
            />
</a>,
          "className": "article",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="article link"
              src="link.svg"
/>,
            "href": undefined,
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "article link",
              "src": "link.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        "type": "div",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="badge"
>
            <img
                        alt="flight badge"
                        src={undefined}
            />
</div>,
          <div
            className="name"
/>,
          <div
            className="type"
/>,
          <div
            className="date"
/>,
          <div
            className="details"
/>,
          <div
            className="id"
/>,
          <div
            className="article"
>
            <a
                        href={undefined}
                        target="_blank"
            >
                        <img
                                    alt="article link"
                                    src="link.svg"
                        />
            </a>
</div>,
        ],
        "className": "LaunchCard",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="flight badge"
              src={undefined}
/>,
            "className": "badge",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "flight badge",
              "src": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "name",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "type",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "date",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "details",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "id",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href={undefined}
              target="_blank"
>
              <img
                            alt="article link"
                            src="link.svg"
              />
</a>,
            "className": "article",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="article link"
                src="link.svg"
/>,
              "href": undefined,
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "article link",
                "src": "link.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
          "type": "div",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
